app/build.gradle

```diff
@@ -17,12 +17,12 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-        }
-    }
+//    buildTypes {
+//        release {
+//            minifyEnabled false
+//            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+//        }
+//    }
```

Comment:
Nitpicky, but do we need to have commented-out stuff still included?

Reply:
No we don't. I'll clean it up a little bit and re-request review, I think there are a few files I created that can be deleted as well. Also need to test and make sure the build fails when it's supposed to.

Reply:
Sounds good